kaitaistruct.py

```diff
@@ -279,57 +351,67 @@ def read_bits_int_le(self, n):
     # Byte arrays
     # ========================================================================
 
-    def read_bytes(self, n):
+    def alignment(self, a):
+        return (a - self.pos()) % a
+
+    def read_bytes(self, n, align=0):
```

Comment:
I see that you're adding new arguments to a few existing functions. In terms of backwards compatibility that's okay, as long as you provide appropriate defaults (which you do, so that's not an issue). But it would be useful to have some context regarding the added arguments - what are they needed for, and how are they meant to be used in code generated by KSC?

About the `align` argument in particular, I assume that is meant to allow reading data that is aligned to a n-byte boundary in the data? The KSY language doesn't have native support for this, so I don't understand how this parameter is meant to be used. If you're interested in implementing native support for alignment, that would be better done in a separate PR (see also the previous discussion about this kind of feature in kaitai-io/kaitai_struct#12).

Reply:
Yes, the code is from that issue. I want to implement it but I am not sure I'll have time and it's useful to have as a service method anyway as there are many use-cases in the wild. That's why that thread is so active.

Reply:
I've written example code below. Should I really make the separate PR? Neither functionality is done on the compiler side right now and it would be a nice demonstration of how it could work so maybe someone, or myself when I have a free time, can pick on it and add them to the compiler.